x-pack/plugins/infra/public/pages/metrics/metrics_explorer/components/chart_context_menu.tsx

```diff
@@ -156,7 +156,7 @@ export const MetricsExplorerChartContextMenu: React.FC<Props> = ({
     ? [
         {
           name: i18n.translate('xpack.infra.metricsExplorer.alerts.createAlertButton', {
-            defaultMessage: 'Create alert',
+            defaultMessage: 'Create threshold rule',
```

Comment:
I think it'd be good if we also changed the i18n labels to reflect the new language as well.  So instead of `xpack.infra.metricsExplorer.alerts.createAlertButton` something like `xpack.infra.metricsExplorer.rules.createRuleButton`.

Reply:
That makes sense. I was hesitant to make any changes over than just the text. Should we create another issue for this and @mgiota's [above](https://github.com/elastic/kibana/pull/103589/files/9ba3ce9ddf27abf120c630c9b32e5831ec90392c#r660929787)?